ios/README.md

```diff
@@ -1,7 +1,4 @@
-![Build status](https://onedrive.visualstudio.com/Design/_apis/build/status/fabric-mobile/fabric-ios-CI-github?branchName=master)
-[![GitHub release](https://img.shields.io/github/release/officedev/ui-fabric-ios.svg)](https://github.com/OfficeDev/ui-fabric-ios/releases/latest)
-
-# [Office UI Fabric for iOS](http://dev.office.com/fabric)
+# Fluent UI for iOS
 
 ##### The iOS UI framework for building experiences for Office and Office 365.
 
```

Comment:
One more instance of "Office UI Fabric" on the line below: "Office UI Fabric for iOS is a native library..."

Reply:
Also looks like there's a section in the Contents for "Install and use Office UI Fabric"?